diff --git a/model/model.py b/model/model.py
index 3a83404..a30c929 100644
--- a/model/model.py
+++ b/model/model.py
@@ -1,5 +1,5 @@
 import torch as pt
-from src.model_operations import StateUpdateLayer, StatePoolLayer, unpack_state_features
+from ..src.model_operations import StateUpdateLayer, StatePoolLayer, unpack_state_features
 
 class Model(pt.nn.Module):
     def __init__(self, config):
diff --git a/src/model_operations.py b/src/model_operations.py
index 3f13d69..797b785 100644
--- a/src/model_operations.py
+++ b/src/model_operations.py
@@ -140,7 +140,7 @@ class StateUpdateLayer(pt.nn.Module):
         ids_nn = ids_topk[:,self.m_nn]
         q = q.requires_grad_()
         p = p.requires_grad_()
-        q, p = checkpoint(self.su.forward, q, p, q[ids_nn], p[ids_nn], D_topk[:,self.m_nn], R_topk[:,self.m_nn])
+        q, p = checkpoint(self.su.forward, q, p, q[ids_nn], p[ids_nn], D_topk[:,self.m_nn], R_topk[:,self.m_nn], use_reentrant=True)
         q[0] = q[0] * 0.0
         p[0] = p[0] * 0.0
         return q, p, ids_topk, D_topk, R_topk
\ No newline at end of file
